src/index.html

```diff
@@ -11,6 +11,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="form_big_bar bar_form_margin"
+      action="#"
+      method="POST"
+      data-qa="big"
+    >
+      <div>
+        <label>
+          <img
+            class="icon-big icon_position"
+            src="images/Search.svg"
+            alt ="search-icon-bg"
```

Comment:
```suggestion
            alt="search-icon-bg"
```

Reply:
also, it's not the best option for alt attribute, imagine users with disabilities, that will hear from screenreader that there is a `search-icon-bg`😅 it's better to just shortly describe image